Bottom: dacd55b95eec3000990a05064321a680cd2dae9e
Top:    605f46b9abd76a88e8e95d32ce99d0aafea6ff06
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-06 02:26:24 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
